include/roaring/roaring_array.h

```diff
@@ -51,9 +51,9 @@ roaring_array_t *ra_create(void);
 bool ra_init_with_capacity(roaring_array_t *new_ra, uint32_t cap);
 
 /**
- * Initialize with default capacity
+ * Initialize with zero capacity
  */
-bool ra_init(roaring_array_t *t);
+void ra_init(roaring_array_t *t);
```

Comment:
Wouldn't this be marked noexcept as well?

Reply:
I'm not entirely sure what you are asking.. That is a C function and cannot be marked as noexcept

Reply:
Oops I got lost.  You're right.  GCC does have a nothrow attribute you can add to C functions but it's not necessary here.